### PR TITLE
Price badge improvements

### PR DESCRIPTION
**Pull request by miikkatu**
This fixes both #1193 and #1392.

### CODE REVIEW

**Comment by seanyesmunt** [comment]
Why the weird padding sizes? Lets just keep it at 5px

File: src/renderer/scss/_gui.scss [hunk 1]
```diff
@@ -252,10 +252,11 @@ p {
 }
 
 .credit-amount {
-  padding: 5px;
```
Comment (seanyesmunt) [inline src/renderer/scss/_gui.scss]: Just keep this and change the font-size. It seems to look fine.

Comment (miikkatu) [reply src/renderer/scss/_gui.scss]: PR is updated now.

**Comment by tzarebczan** [comment]
Thanks @miikkatu ! Got your email, will follow up with a tip shortly.

**Pull request merged**